web/src/beta/features/Navbar/types.ts

```diff
@@ -0,0 +1,22 @@
+import { ProjectType } from "@reearth/types";
+
+export type User = {
+  name: string;
+};
+
+export type Workspace = {
+  id?: string;
+  name?: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Please don't use this unless absolutely necessary. 
Isn't the members just simply going to be `Array<User>` or something?

Reply:
I used Uknown cause using User lead to type check errors in other places

Reply:
and also because the type should match with the type existed in state and if change both may affect the classic project so I did the minimum change to avoid affect reearth classic

Reply:
Currently, this navbar should only be used in beta (the editor), so I am not sure why you would be getting type errors, etc connected with classic

Reply:
yes but state is common between both beta and classic and any change in it may affect other fields

Reply:
So can you figure out the state's type and use that here. I only said User because there is a User type in the file already so I thought maybe.
But as you say, you got a type error. So whatever the needed type is please make it that here.

Reply:
dear , I am using same state type but in this file there was an eslint warning , that I couldn't use any explicitly while it is not existed instate

Reply:
Lets talk about this after daily scrum. I just want to make sure we both understand one another 👍🏻